testdata/ingest_external

```diff
@@ -996,3 +996,54 @@ p1_a@1: (va, .)
 c@1: (vc, .)
 a@1: (va, .)
 .
+
+# Tests with range keys.
+
+reset
+----
+
+build-remote points-and-ranges.sst
+set a@1 va
+set b@1 vb
+set c@1 vc
+set d@1 vc
+range-key-set b c1 @2 vrange
+----
+
+ingest-external
+points-and-ranges.sst bounds=(a,e) has-range-keys
+----
+
+iter
+first
+next
+next
+next
+next
+next
+----
+a@1: (va, .)
+b: (., [b-"c1") @2=vrange UPDATED)
+b@1: (vb, [b-"c1") @2=vrange)
+c@1: (vc, [b-"c1") @2=vrange)
+d@1: (vc, . UPDATED)
+.
+
+ingest-external
+points-and-ranges.sst synthetic-prefix=x_ bounds=(x_a,x_d) has-range-keys
```

Comment:
Let's try adding a case where the last key is the rangedel's end key, and yet we're at a loose bound? So something like `[a, c2)` for the vsst bounds.